WalletWasabi.Tests/UnitTests/StartWasabiOnSystemStartupTests.cs

```diff
@@ -8,22 +9,22 @@ namespace WalletWasabi.Tests.UnitTests
 	public class StartWasabiOnSystemStartupTests
 	{
 		[Fact]
-		public void ModifyStartupOnDifferentSystemsTest()
+		public async Task ModifyStartupOnDifferentSystemsTestAsync()
 		{
 			if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
 			{
-				StartupHelper.ModifyStartupSetting(true);
-				StartupHelper.ModifyStartupSetting(false);
+				await StartupHelper.ModifyStartupSettingAsync(true);
+				await StartupHelper.ModifyStartupSettingAsync(false);
 			}
 			else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
 			{
-				Assert.Throws<NotImplementedException>(() => StartupHelper.ModifyStartupSetting(true));
-				Assert.Throws<NotImplementedException>(() => StartupHelper.ModifyStartupSetting(false));
+				await Assert.ThrowsAsync<NotImplementedException>(async () => await StartupHelper.ModifyStartupSettingAsync(true));
+				await Assert.ThrowsAsync<NotImplementedException>(async () => await StartupHelper.ModifyStartupSettingAsync(false));
 			}
 			else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
 			{
-				Assert.Throws<NotImplementedException>(() => StartupHelper.ModifyStartupSetting(true));
-				Assert.Throws<NotImplementedException>(() => StartupHelper.ModifyStartupSetting(false));
+				await StartupHelper.ModifyStartupSettingAsync(true);
+				await StartupHelper.ModifyStartupSettingAsync(false);
```

Comment:
Can this test disable WW to be started on startup on a dev computer?

Reply:
Unfortunately it can. We couldn't think out any good solution to prevent this.

Reply:
I agree, but if there would be something it can be done in next PR.